services/user.py

```diff
@@ -0,0 +1,37 @@
+from db.models import User
+
+
+def create_user(username: str, password: str, email: str = None,
+                first_name: str = None, last_name: str = None) -> User:
+    user = User.objects.create_user(username=username, password=password)
```

Comment:
The `create_user` function should validate the input data to ensure that the username and email are unique. Consider using Django's built-in validation or manually checking for existing users with the same username or email before creating a new user.